packages/Card/src/Card.styles.js

```diff
@@ -40,7 +40,7 @@ const activeStyles = `
 export const cardStyles = styled.div(
   ({ size, isActive, isFullWidth }) => css`
     background: ${tokens.color.white};
-    border: ${tokens.border.color};
+    border: 1px ${tokens.color.blackLighten60};
```

Comment:
There's still no border rendered if you don't include the `solid` part.

`border: 1px solid ${tokens.color.blackLighten60};`